iceoryx_posh/source/popo/ports/server_port_data.cpp

```diff
@@ -23,13 +23,20 @@ namespace popo
 {
 ServerPortData::ServerPortData(const capro::ServiceDescription& serviceDescription,
                                const RuntimeName_t& runtimeName,
-                               const NodeName_t& nodeName,
+                               const ServerOptions& serverOptions,
                                mepoo::MemoryManager* const memoryManager,
                                const mepoo::MemoryInfo& memoryInfo) noexcept
-    : BasePortData(serviceDescription, runtimeName, nodeName)
-    , m_chunkSenderData(memoryManager, SERVER_SUBSCRIBER_POLICY, 0, memoryInfo)
-    , m_chunkReceiverData(cxx::VariantQueueTypes::FiFo_MultiProducerSingleConsumer, SERVER_PUBLISHER_POLICY)
+    : BasePortData(serviceDescription, runtimeName, serverOptions.nodeName)
+    , m_chunkSenderData(
+          memoryManager, static_cast<SubscriberTooSlowPolicy>(serverOptions.clientTooSlowPolicy), 0, memoryInfo)
```

Comment:
Is it possible that `clientTooSlowPolicy` contains an integer which is not convertable to `SubscriberTooSlowPolicy`?
Just asking - not reviewing ;)

Reply:
This is just a stop gap solution. `SubscriberTooSlowPolicy` will be removed and replaced with `ConsumerTooSlowPolicy`, the type of `clientTooSlowPolicy`. Since that change affects quite many files, it is deferred to a later point in time. See also the `todo` in  `port_queue_policies.hpp`

Reply:
I was wondering what the `0` means in this line and realized it is actually a `nullptr` and signals no memory manager set. Could you maybe introduce `constexpr mepoo::MemoryManager* INVALID_MEMORY_MANAGER = nullptr` in `ServerPortData` and use it here.

Otherwise other reviewers have no clue what the `0` means.

Reply:
@elfenpiff you are wrong and by being wrong you showed why magic numbers should be avoided. This is the history request and I will add a constexpr for this

Reply:
@elBoberido I just wanted to check if you are paying attention ;)